### DISCUSSION TITLE
How to share email pasword auth between native and web layers?

### DISCUSSION THREAD

**Discussion by niklv** [capawesome-team/capacitor-firebase#637] [Q&A]
There are no such thing in docs.

**Comment by robingenz** [comment]
I guess that's not possible. Also, this shouldn't be necessary. You don't need any native SDKs for password sign-in.

**Reply by niklv** [reply]
But `firebase-js-sdk` stores auth tokens in indexeddb. And according to https://capacitorjs.com/docs/guides/storage#why-cant-i-just-use-localstorage-or-indexeddb OS can reclaim storage. And user will be logged out.

**Reply by robingenz** [reply]
Usually this rarely happens, but if this is a problem for you, you should only use the native Firebase SDK instead of the Firebase JS SDK.
